testfiles/PlantLoadProfileCooling.idf

```diff
@@ -0,0 +1,398 @@
+    Demand Load Profile 1 Outlet Node,  !- Component 1 Outlet Node Name
+    ACTIVE;                  !- Component 1 Branch Control Type
+
+  LoadProfile:Plant,
+    Load Profile 1,          !- Name
+    Demand Load Profile 1 Inlet Node,  !- Inlet Node Name
+    Demand Load Profile 1 Outlet Node,  !- Outlet Node Name
+    Load Profile 1 Load Schedule,  !- Load Schedule Name
+    0.002,                   !- Peak Flow Rate {m3/s}
+    Load Profile 1 Flow Frac Schedule;  !- Flow Rate Fraction Schedule Name
+
+  Branch,
+    Demand Outlet Branch,    !- Name
+    0,                       !- Maximum Flow Rate {m3/s}
+    ,                        !- Pressure Drop Curve Name
+    Pipe:Adiabatic,          !- Component 1 Object Type
+    Demand Outlet Pipe,      !- Component 1 Name
+    Demand Load Profile-Pipe Node,  !- Component 1 Inlet Node Name
+    Demand Outlet Node,      !- Component 1 Outlet Node Name
+    PASSIVE;                 !- Component 1 Branch Control Type
+
+  Pipe:Adiabatic,
+    Demand Outlet Pipe,      !- Name
+    Demand Load Profile-Pipe Node,  !- Inlet Node Name
+    Demand Outlet Node;      !- Outlet Node Name
+
+  ScheduleTypeLimits,
+    Any Number;              !- Name
+
+  ScheduleTypeLimits,
+    On/Off,                  !- Name
+    0,                       !- Lower Limit Value
+    1,                       !- Upper Limit Value
+    DISCRETE;                !- Numeric Type
+
+  Schedule:Compact,
+    Main Loop Temp Sch,      !- Name
+    Any Number,              !- Schedule Type Limits Name
+    THROUGH: 12/31,          !- Field 1
+    FOR: AllDays,            !- Field 2
+    UNTIL: 24:00,7.22;       !- Field 3
+
+  Schedule:Compact,
+    AlwaysOnSchedule,        !- Name
+    On/Off,                  !- Schedule Type Limits Name
+    THROUGH: 12/31,          !- Field 1
+    FOR: AllDays,            !- Field 2
+    UNTIL: 24:00,1;          !- Field 3
+
+  Schedule:Compact,
+    Load Profile 1 Load Schedule,  !- Name
+    Any Number,              !- Schedule Type Limits Name
+    THROUGH: 12/31,          !- Field 1
+    FOR: AllDays,            !- Field 2
+    UNTIL: 4:00,-8000,        !- Field 3
+    UNTIL: 8:00,-6000,        !- Field 5
+    UNTIL: 9:00,0,           !- Field 7
+    UNTIL: 12:00,-6000,       !- Field 9
+    UNTIL: 24:00,-10000;      !- Field 11
+
+  Schedule:Compact,
+    Load Profile 1 Flow Frac Schedule,  !- Name
+    Any Number,              !- Schedule Type Limits Name
+    THROUGH: 12/31,          !- Field 1
+    FOR: AllDays,            !- Field 2
+    UNTIL: 24:00,1.0;        !- Field 3
+
+  Output:VariableDictionary,Regular;
+
+  Output:Variable,*,System Node Temperature,Hourly;
+
+  Output:Variable,*,System Node Mass Flow Rate,Hourly;
+
+  Output:Variable,*,Plant Load Profile Mass Flow Rate,Hourly;
+
+  Output:Variable,*,Plant Load Profile Heat Transfer Rate,Hourly;
+
+  Output:Variable,*,Plant Load Profile Heat Transfer Energy,Hourly;
+
+  Output:Variable,*,District Cooling Hot Water Rate,Hourly;
```

Comment:
@Myoldmopar This output variable and the next should say "Chilled Water" instead of "Hot Water".  I was expecting PlantLoadProfileCooling.idf and PlantLoadProfileCoolingReturnReset.idf to differ only in the chilled water loop setpoint controls.  Would you be open to redoing PantLoadProfileCooling by copying PlantLoadProfileCoolingReturnReset and simply changing the setpoint manager to supply temp control?